server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -314,6 +397,18 @@ Response createOrUpdate(CreateOrUpdateSecretRequestV2 request, String name) thro
     return mapper.readValue(httpResponse.body().byteStream(), new TypeReference<List<String>>(){});
   }
 
+  List<SecretDetailResponseV2> listVersions(String name, int newestIdx, int oldestIdx)
```

Comment:
I'll rename these parameters to versionIdx and numVersions in the next commit, for clarity.